src/cli/handlers/interrupts.py

```
@@ -24,6 +26,10 @@
 class InterruptHandler:
     """Handles LangGraph interrupts and collects user input for resume."""
 
+    def __init__(self, session):
```

Ruff (0.14.6)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)